z2/src/chain/node.rs

```diff
@@ -515,17 +515,29 @@ impl ChainNode {
         let role_name = self.role.to_string();
         let eth_chain_id = self.eth_chain_id.to_string();
         let bootstrap_public_ip = selected_bootstrap.machine.external_address;
+        let enabled_apis = if self.role == NodeRole::Api {
+            // Enable all APIs, except `admin_` for API nodes.
+            json!(["erigon", "eth", "net", "ots", "trace", "txpool", "web3", "zilliqa"])
+        } else {
+            // Only enable `eth_blockNumber` for other nodes.
+            json!([
+                {
+                    "namespace": "eth",
+                    "apis": ["blockNumber"],
+                },
+            ])
+        };
```

Comment:
Does it make sense for us to enable all APIs on API nodes and only have `eth_blockNumber` enabled on other nodes?

Reply:
Yeah possibly - I guess I was just being as conservative as possible but its true that all `eth_` APIs are _probably_ safe?

Reply:
Most of them come in handy when testing, we can always remove later :)